KosherCocoa/Library/Core/Solar/KCAstronomicalCalendar.h

```diff
@@ -12,23 +12,25 @@
 
 #import "KCGeoLocation.h"
 #import "KCSunriseAndSunsetCalculator.h"
+#import "KCNOAACalculator.h"
+#import "KCAstronomicalCalculator.h"
 
-/** 
- A class that calculates various sunrise and sunset values. 
+/**
+ A class that calculates various sunrise and sunset values.
  */
 NS_SWIFT_NAME(AstronomicalCalendar)
 @interface KCAstronomicalCalendar : NSObject
 
  // MARK: - Configuring the Calendar
 
 /**
- * The calculator to use. 
+ * The calculator to use.
  *
- *  In KosherCocoa, it's always a KCSunriseAndSunsetCalculator. The
- *  original Java library offered several calculator options.
+ *  The default calculator in KosherCocoa is the KCSunriseAndSunsetCalculator which is based on the SunTimesCalculator in KosherJava.
+ *  However, you can use the KCNOAACalculator as well.
  */
 
-@property (nonatomic, strong, nonnull) KCSunriseAndSunsetCalculator *astronomicalCalculator;
+@property (nonatomic, strong, nonnull) typeof(KCAstronomicalCalculator) *astronomicalCalculator;
```

Comment:
I tried using `NSObject<KCAstronomicalCalculator>` but it kept giving me errors that were above my head to fix. typeof just worked, so that is what I used, I forget how I came to that conclusion. You can feel free to change it if you want, but "Don't fix what's not broken!", y'know.